CONTRIBUTING.md

```diff
@@ -1,62 +1,172 @@
+
+```bash
+npm run format
+```
+
+9.  Once you're happy with your changes, add and commit them to your branch,
+    then push the branch to your fork.
 
-- A **Contributor** is any individual creating or commenting on an issue or pull request.
-- A **Collaborator** is a subset of contributors who have been given write access to the repository.
+        ```bash
+        cd ~/nodejs.org
+        git add .
+        git commit -m "some message"
+        git push -u origin name-of-your-branch
+        ```
 
-## Logging Issues
+10. Create a Pull Request. See [PR Policy](#pull-request-policy)
 
-Log an issue for any question or problem you might have. When in doubt, log an issue.
-Any additional policies about what to include will be provided in the responses. The only
-exception is security disclosures which should be sent privately.
+> **Note**: Go through our [Commit](#commit-guidelines) and [Pull Request](#pull-request-policy) guidelines outlined below.
 
-Collaborators may direct you to another repository, ask for additional clarifications, and
-add appropriate metadata before the issue is addressed.
+### CLI Options
 
-Please be courteous, respectful, and every participant is expected to follow the
-project's Code of Conduct.
+#### Serve/Build Options
 
-## Contributions
+- `npm run serve` runs Next.js's Local Development Server, listening by default on `http://localhost:3000/`.
+- `npm run build` builds the Application on Production mode. The output is by default within `.next` folder.
+- `npm run export` exports the website from the `.next` into a fully static website. The output is by default within `build` folder.
+  - This is what it's used to deploy the website on our current Node.js servers.
+- `npm run start` starts a web server running serving the built content from `npm run build`
 
-Any change to resources in this repository must be through pull requests.
+#### Other CLI options
+
+We also offer other commands that offer you assistance during your local development
+
+- `npm run lint` runs the linter for all the js files.
+  - `npm run lint:fix` attempts to fix any linting errors
+- `npm run prettier` runs the prettier for all the js files.
+  - `npm run prettier:fix` attempts to fix any style errors
+- `npm run format` formats and fixes the whole codebase
+- `npm run scripts:release-post` generates a release post for the current release
+  - **Usage:** `npm run scripts:release-post -- --version=vXX.X.X --force`
+
+## Commit Guidelines
+
+This project follows the [Conventional Commits][] specification.
+
+Commits should be signed. You can read more about [Commit Signing][] here.
+
+### Commit Message guidelines
+
+- Commit messages must include a "type"
+  > build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test
+- Commit messages **must** start with a capital letter
+- Commit messages **must not** end with a period `.`
+
+## Pull Request Policy
+
+### Before merging
```

Comment:
Maybe also include "Commits should be signed." warning here?

Reply:
We added this on "Commit Guidelines" already.